pyte/screens.py

```diff
@@ -348,7 +335,6 @@ def set_mode(self, *modes, **kwargs):
         # When DECOLM mode is set, the screen is erased and the cursor
         # moves to the home position.
         if mo.DECCOLM in modes:
-            self.resize(columns=132)
```

Comment:
This was needed, see DECCOLM section int [VT220 manual](http://vt100.net/docs/vt220-rm/chapter4.html).